tests/workflows/try_node_execute_wrapped_node/tests/test_workflow.py

```diff
@@ -0,0 +1,27 @@
+from uuid import UUID
+
+from tests.workflows.try_node_execute_wrapped_node.workflow import TryNodeExecuteWrappedNodeWorkflow, WrappedNode
+
+
+def test_execute_try_node_wrapped_node_by_id():
+    """
+    Test that we can execute the inner wrapped node of a Try Node adornment
+    using run_node and verify it executes with the correct ID.
+    """
+
+    # GIVEN a workflow with a Try Node wrapped node
+    workflow = TryNodeExecuteWrappedNodeWorkflow()
+
+    # AND we get the inner wrapped node's ID and class
+    inner_node_id = WrappedNode.__wrapped_node__.__id__
+    inner_node_class = WrappedNode.__wrapped_node__
+    assert isinstance(inner_node_id, UUID)
+
+    events = list(workflow.run_node(inner_node_class))
```

Comment:
Update the `run_node` method to accept either a node id, node class, or a string that represents `f"{node_class.__module__}.{node_class.__name__}"` so we could migrate the logic that we have in the workflow's `executor.py`

Reply:
DevinAI iterate on my comment above